### PR TITLE
pkgdown improvements

### LINKED ISSUE #3883: pkgdown todo list
Thread to gather issues spotted in new pkgdown website: https://rdatatable.gitlab.io/data.table

- [ ] missing current devel news at [changelog](https://rdatatable.gitlab.io/data.table/news/index.html) page (fixed by #3888)
- [x] links to manual (e.g. fwrite, fread) does not point to expected location (fixed by #3882)
- [ ] links mentioned above works fine in a website but not on README.md in github, we could put an absolute urls there maybe  - (fixed by #3888)
- [ ] `NA` title in a [contributing page](https://rdatatable.gitlab.io/data.table/CONTRIBUTING.html) that is linked from right side panel (fixed by #3888)
- [ ] Cheatsheet thumbnail appears broken - (fixed by #3888)

### PR DESCRIPTION
**Pull request by jangorecki**
closes #3883

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# data.table <a href="http://r-datatable.com"><img src="logo.png" align="right" height="140" /></a>
+# data.table <a href="https://rdatatable.gitlab.io/data.table"><img src="https://raw.githubusercontent.com/Rdatatable/data.table/master/.graphics/logo.png" align="right" height="140" /></a>
```
Comment (mattdowle) [inline README.md]: @jangorecki Shall I redirect r-datatable.com to rdatatable.gitlab.io/data.table now?  The idea was that r-datatable.com was the canonical URL to publicize and link, then I can just change where it forwards to, like now.  If so, the logo can be left linking to r-datatable.com ?

Comment (jangorecki) [reply README.md]: yes, I think we can redirect http://r-datatable.com to rdatatable.gitlab.io/data.table

**Comment by jangorecki** [comment]
cheatsheet thumbnail will get updated automatically once https://github.com/rstudio/cheatsheets/pull/102 merged, I removed png from our repo as we can keep it together in the same place where pdf is located

File: README.md (hunk 1 above)
Comment (mattdowle) [reply README.md]: Done.

**Pull request merged**